thrift/thrift0.17/src/test/java/com/linecorp/armeria/internal/server/thrift/ThriftDocServicePluginTest.java

```diff
@@ -222,14 +223,16 @@ private static List<String> methods(Map<String, ServiceInfo> services) {
 
     @Test
     void testNewServiceInfo() {
+        final EntryBuilder builder = new EntryBuilder(FooService.class);
+        builder.endpoint(EndpointInfo.builder("*", "/foo")
+                                     .fragment("a").defaultFormat(ThriftSerializationFormats.BINARY)
+                                     .build());
+        builder.endpoint(EndpointInfo.builder("*", "/debug/foo")
```

Comment:
What do you think of also adding an integration test to `:it:thrift-fullcamel` to ensure that `DocService` runs a proper specification with the Thrift fullcamel option?

Reply:
OK, I will give it a try.